safe_app_electron_quick_start/README.md

```diff
@@ -83,10 +55,15 @@ $ npm start
 You should see a "Hello SAFE Network!" message in our app's window and an empty list of trips. We are now ready to start creating the code to be able to store the planned trips into the SAFE Network.
 
 ## Import the SAFE API
-The application will interact with the SAFE Network using the `safe-node-app` package, we therefore need to add it as a dependency in our package:
+The application will interact with the SAFE Network using the `safe-node-app` package, we therefore need to add it as a dependency in our package.
+
+As mentioned above in the [pre-requisites](#pre-requisites) section, it's recommended to use the browser built for mock routing for this tutorial, therefore we need to make sure to install the SAFE libraries required to connect to mock routing as well. We do this by setting the `NODE_ENV` environment variable specifically during installation of the `safe-node-app` package:
 ```bash
-$ npm install @maidsafe/safe-node-app --save
+$ NODE_ENV=dev npm install @maidsafe/safe-node-app --save
```

Comment:
Soon to be not needed to set env var once this [PR](https://github.com/maidsafe/safe_app_nodejs/pull/295) is merged.

Reply:
I just added a comment to that PR, and even that I would remove the NODE_ENV=dev from the tutorial, I'm proposing to make the change in that other PR to backward compatible.
Either way, analogous to previous comment, I think we should keep this as is until we release new version of safe-app-nodejs